toptal/picasso
# feat(ScrollMenu): increase max height

[FX-2349]

### Description
As confirmed (please check ticket), Picasso **Autocomplete** should have a bigger max height, so the scrollbar is shown only for a number of items > 8.

### How to test
- go to: https://picasso.toptal.net/FX-2349-increase-max-height-for-scroll-menu/?path=/story/picasso-readme--readme
- check following components:
  - `Page.Autocomplete`
  - `Autocomplete`

### Screenshots
**Page.Autocomplete (no scrollbar): 4 items on the left, 8 items on the right**
| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img src="https://user-images.githubusercontent.com/4816942/145555499-0823ab2f-f33f-4b8f-82ab-70152dca5e5e.png" width="600" /> | <img src="https://user-images.githubusercontent.com/4816942/145555602-b4d3726f-839e-4e25-85be-c41e45f8ccde.png" width="600" /> |

**Page.Autocomplete: 10 items**
| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img src="https://user-images.githubusercontent.com/4816942/145555190-1324910b-9582-4728-8711-790f8da7f384.png" width="600" /> | <img src="https://user-images.githubusercontent.com/4816942/145555827-469261d6-0e88-4653-8a3e-245fafba28d1.png" width="600" /> |

**Autocomplete**
| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img src="https://user-images.githubusercontent.com/4816942/145554203-a548a076-11de-43f0-a7f7-f6486e626f71.png" width="600" /> | <img src="https://user-images.githubusercontent.com/4816942/145554222-9d4bc37f-cb5a-47f4-aa83-d67c58841c97.png" width="600" /> |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-2349]: https://toptal-core.atlassian.net/browse/FX-2349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
